src/drivers/drv_sensor.h

```diff
@@ -145,6 +145,7 @@
 #define DRV_OSD_DEVTYPE_ATXXXX          0x6b
 #define DRV_ACC_DEVTYPE_BMI085          0x6C
 #define DRV_GYR_DEVTYPE_BMI085          0x6D
+#define DRV_DIFF_PRESS_DEVTYPE_ASP5033  0x6D
```

Comment:
This conflicts now.

Reply:
How could I change this the working address of ASP5033 is 0x6D? Could you give me so advice, please?

Reply:
Pick one that is still available in the file. I'm confused.

Reply:
Thank you very much!